wiki/Interface/en.md

```diff
@@ -1,227 +1,297 @@
--   A "Not yet uploaded" means it's your created beatmap but you didn't upload it yet.
-
-| colspan=2|Ranking Table       |
-|-------------------------------|
-| Name                          |
-| Local Ranking                 |
-| Country Ranking               |
-| Global Ranking                |
-| Global Ranking (Selected Mod) |
-| Friend Ranking                |
-
-Click the <img src="Word bubble.jpg" title="fig:Word bubble.jpg" alt="Word bubble.jpg" width="15" />word bubble icon to call up the **Quick Web Access** screen for the selected beatmap:
-
--   Press 1 or click the "Beatmap Listing/Scores" button and your default Internet browser will attempt to display the Beatmap Listing and score page of the beatmap set the selected beatmap belongs to.
--   Press 2 or click "Beatmap Topic" and your default Internet browser will attempt to display the forum thread of the beatmap set the selected beatmap belongs to.
--   Press 3 or click "Quick Reply" and your default Internet browser will attempt to open a page where you can compose a reply to the forum thread of the beatmap set the selected beatmap belongs to.
--   Press 4 or Esc or click "Cancel" to return to the Song Selection Screen.
+![](img/Soloplay10.jpeg)
+
+ A variety of things can appear in this space:
+
+-   A "Not Submitted" box denotes a beatmap that has not been uploaded to the osu! site using the Beatmap Submission System.
+-   An "Update to latest version" box appears if there is a new version of the beatmap available for download. Click on the button to update.
+    -   **Note:** Once you update the beatmap, it cannot be reversed. If you want to preserve the older version for some reason (say, to keep scores), then do not update. 
+-   A "Latest pending version" box appears means that the beatmap has been uploaded to the osu!website but is not ranked yet.
+-   If replays matching the view setting of the beatmap exist, they will be displayed instead of a box denoting the ranked/played status of the beatmap. This is shown in the above picture.
+    - Under public rankings (e.g. Global, Friends, etc.), your high score will be shown at the bottom, as well as your rank on the leaderboard. 
+-   A "No records set!" box means that there are no replays for the current view setting (this is typically seen in the Local view setting if you just downloaded or edited the beatmap). 
+    -   Note: Scores for Multi are not counted as records.
+
+These are the view settings:
+
+- Local Ranking
+- Country Ranking\*
+- Global Ranking
+- Global Ranking (Selected Mods)\*
+- Friend Ranking\*
+
+Settings with an asterisk require you to be an [osu!supporter](/wiki/osu!supporter/) to access them.
+
+Click the word bubble icon to call up the **Quick Web Access** screen for the selected beatmap:
+
+-   Press `1` or click the "Beatmap Listing/Scores" button and your default internet browser will pull up the Beatmap Listing and score page of the beatmap set the selected beatmap belongs to.
```

Comment:
use code for the button names
`` `Beatmap Listing/Scores` ``
do the same below